build.sbt

```
@@ -55,6 +57,34 @@ ThisBuild / githubWorkflowPublish := Seq(
   )
 )
 
+// publish website
+
+ThisBuild / githubWorkflowAddedJobs += WorkflowJob(
+  id = "site",
+  name = "Deploy site",
+  needs = List("publish"),
+  javas = (ThisBuild / githubWorkflowJavaVersions).value.toList,
+  scalas = (ThisBuild / scalaVersion).value :: Nil,
+  cond = """
+  | needs.build.result == 'success' &&
+  | (needs.publish.result == 'success' && startsWith(github.ref, 'refs/tags/v'))
+  """.stripMargin.trim.linesIterator.mkString.some,
```

Ah, the suggestion from above should have been here:
```suggestion
 cons = "startsWith(github.ref, 'refs/tags/v')"
```